bitbots_ros_control/launch/ros_control.launch

```diff
@@ -13,7 +15,21 @@
         <rosparam param="torquelessMode" ns="/ros_control">false</rosparam>
     </group>
 
-    <node pkg="bitbots_ros_control" type="dynamixel_controller_manager_node" name="ros_control" output="screen"/> 
+    <group if="$(arg onlyImu)">
+        <rosparam param="onlyImu" ns="/ros_control">true</rosparam>
+    </group>
+    <group unless="$(arg onlyImu)">
+        <rosparam param="onlyImu" ns="/ros_control">false</rosparam>
+    </group>
+
+    <group if="$(arg onlyPressure)">
+        <rosparam param="onlyPressure" ns="/ros_control">true</rosparam>
+    </group>
+    <group unless="$(arg onlyPressure)">
+        <rosparam param="onlyPressure" ns="/ros_control">false</rosparam>
+    </group>
```

Comment:
This could be changed to the one-line if/unless syntax (or maybe `$(arg ...)` can even be used directly as the param value.